### DISCUSSION TITLE
Welcome to www Discussions!

### DISCUSSION THREAD

**Discussion by mjang** [writethedocs/www#1596] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mjang** [comment]
I wanted to start a discussion based on a thread in the WtD Slack Channel. The thread is a discussion on how documentarians present themselves in the resumes/CVs/LinkedIn profiles. 

Question for y'all: What are your "lessons learned" from the thread.

**Reply by mjang** [reply]
For me, the key question for documentarians:

- Do you want to specialize in some technical expertise? Examples:
  - Code
  - APIs
  - Databases
  - DevOps
  - Security

- Or would you rather work as a more "Generalist" writer?

**Reply by StarfallProjects** [reply]
Is it ok for me to add a few more context notes on the thread? I found the original question & discussion interesting as well.

- Started with someone asking why writers tend not to give tech skills (e.g. programming knowledge) on CVs, in their experience recruiting
- Followed by writers providing some reasons they don't (expectation of learning on the job so long as the writers has sufficient interest, wariness of noting skills they might be assessed on when they don't have a high level, writing & docs tooling arguably being the core skills)

This initial exchange led to the broader discussion around the place of specific tech skills, value of specialising, what skills might be expected (or not), and so on. I think the number of participants and replies (over 100), and the perspectives from several people involved in hiring, shows just how important it is for tech writers to think about this when making career/learning/resume choices.

**Reply by StarfallProjects** [reply]
Now to respond to @mjang question: I've enjoyed being a generalist so far, and I don't think it's done me any harm (if anything it possibly fosters user empathy and challenges me as a writer) but I'm at the point where I think I'd like to specialise - partly for possible career benefits, but also for personal interest: I'd like to get _properly_ good at some area of tech.

That said, I think it's important to keep sight of the key skills (writing well, structuring information etc.), and I do think it helps if you can manage your own tooling to a great extent (so basic web dev skills are likely needed, regardless of specialism - HTML & CSS, as well as some idea of hosting & deployment, are relevant whether you're working with Flare or a modern SSG or something else). I'd probably add git to the list of key skills, assuming you want to use docs like code and/or work closer to the dev end of software docs. And that's already a lot! 

As for what to specialise in . . . I'm nearing the end of a very enjoyable API docs project, and am also finding an enjoyable challenge contributing to docs on a CI tool. So while I'm trying to choose a specialty, there's a high chance I'll keep doing a variety of things for a while . . . 

I'd be really interested to hear from people who do have a specialty: how did you choose it?

**Reply by feorlen** [reply]
My specialization as a tech writer is directly from previous work in similar areas (as a dev, but also support, QA, etc.) I wrote documentation in those other roles, but it was not my primary task. (Sometimes I did it because docs were needed, despite that not being technically my job.) I started my career in server-side applications, it was relatively easy to shift to cloud technologies as they became more common.

So I have a lot of domain knowledge, but less of what might be thought of as traditional tech writing expertise. I'm self-taught as a writer and use most of the same tools as my organization's developers. I'm aware of "tech writer tech" but don't use much of it.

This has worked well for tech startups, generally the same sort of company I might have previously joined as an engineer. The documentation I produce in my current role is just as much "Minimum Viable Product" and iterative as the software I'm writing about. I can solve many of my own problems and reduce the need for hands-on assistance from an eng SME. I also advocate for the user just as much as when I was in support.

One thing about the transition however, it very much felt like a one way trip. I have historically moved in and out of dev roles. Each non-coding role made it more difficult to go back to dev. Yet larger companies with established documentation teams were also less interested, because I didn't know the tech writer tech.

**Comment by JodyWinter** [comment]
Hi everyone. I'm based in Auckland, New Zealand. I've been a tech writer since 2004. I'd been working in technology since 1996 as a project coordinator, but went back to study in 2003 and earned a post-grad diploma in technical communication. I have been a generalist mostly. I started my career writing standard operating procedures and disaster recovery procedures for technical audiences. Since 2010 I have written mostly online help and user guides for end-users. I use Author-it as my CMS, Camtasia for videos, Audacity and AWS Polly for sound bites, SnagIt for screen captures.

Last year I decided I wanted to specialise. I have been learning how to document REST APIs using the OpenAPI Spec. This has introduced me to docs-as-code and a host of other methodologies and tools. For example, GitHub, Markdown, Stoplight, SwaggerUI and Postman. 
I already had pretty decent HTML and CSS skills but would like to develop these more. Eventually, I want to be a specialist in building developer docs portals.

The biggest challenge for me is to get experience. I got lucky and was able to work on a project for a Swedish company April - August this year thanks to pure luck. I am now looking for another practice project before I go out there and charge money for my expertise. I just put a call out on LinkedIn, I have a [blog](https://jodywinter.com/) that documents my experience and goals. I have reached out to open source projects... But I'm finding it really hard to crack into this field.

**Reply by swapnilogale** [reply]
> Hi everyone. I'm based in Auckland, New Zealand. I've been a tech writer since 2004. I'd been working in technology since 1996 as a project coordinator, but went back to study in 2003 and earned a post-grad diploma in technical communication. I have been a generalist mostly. I started my career writing standard operating procedures and disaster recovery procedures for technical audiences. Since 2010 I have written mostly online help and user guides for end-users. I use Author-it as my CMS, Camtasia for videos, Audacity and AWS Polly for sound bites, SnagIt for screen captures.
> 
> Last year I decided I wanted to specialise. I have been learning how to document REST APIs using the OpenAPI Spec. This has introduced me to docs-as-code and a host of other methodologies and tools. For example, GitHub, Markdown, Stoplight, SwaggerUI and Postman. I already had pretty decent HTML and CSS skills but would like to develop these more. Eventually, I want to be a specialist in building developer docs portals.
> 
> The biggest challenge for me is to get experience. I got lucky and was able to work on a project for a Swedish company April - August this year thanks to pure luck. I am now looking for another practice project before I go out there and charge money for my expertise. I just put a call out on LinkedIn, I have a [blog](https://jodywinter.com/) that documents my experience and goals. I have reached out to open source projects... But I'm finding it really hard to crack into this field.

Hello @JodyWinter and nice to see you here. If you are on the [Write the Docs Slack](https://www.writethedocs.org/slack/), I'd recommend checking out the #job-posts-only and #open-source channels. A couple of other places to check out would be [Working in Content](https://workingincontent.com/), and [Content Writing Jobs](https://contentwritingjobs.com/).

**Reply by JodyWinter** [reply]
Hi @swapnilogale - nice to see you too! Thanks for the tips! I will check them all out. Much appreciated.
